.github/workflows/release.yml

```diff
@@ -287,4 +287,4 @@ jobs:
                       assets+=("-a" "$asset/$asset")
                   done
                   tag_name="${{ env.VERSION }}"
-                  echo "$body" | hub release create --draft "${assets[@]}" -F "-" "$tag_name"
+                  echo "$body" | gh release create --draft "${assets[@]}" -F "-" "$tag_name"
```

Comment:
I think we might need to update the syntax for the assets here, I'm getting the following when running this locally:
`unknown shorthand flag: 'a' in -a`

Reply:
Good catch. I think I've fixed this in 06f6ef6e5f44314cfb98c23407f7a35018e75302. I uploaded a test release with assets to my fork of Lighthouse successfully. I'll also try a tree-states release today.

Reply:
Nice!!